pkg/skaffold/deploy/kustomize.go

```diff
@@ -82,9 +82,10 @@ type secretGenerator struct {
 type KustomizeDeployer struct {
 	*latest.KustomizeDeploy
 
-	kubectl            deploy.CLI
-	insecureRegistries map[string]bool
-	BuildArgs          []string
+	kubectl              deploy.CLI
+	insecureRegistries   map[string]bool
+	debugHelpersRegistry string
```

Comment:
Ditto to the `KubectlDeployer`'s `debugHelpersRegistry`: this seems out of place, and indicates that we have some difficulty communicating information to the debug transforms.

Reply:
Same as in `kubectl.go`